src/shared/jql/simpleJqlParser.ts

```
@@ -0,0 +1,423 @@
+        value = stripQuotes(value);
+        return { type: 'condition', field, op: '=', value };
+      }
+      case isKeyword(op, 'not'): {
+        const nextToken = tokens[pos];
+        if (nextToken != 'in') {
+          throw new Error(`Expected in to get "not in", but got "${nextToken}"`);
+        }
+        pos++;
+        op = 'not in';
+      }
+      case isKeyword(op, 'in') || op.toLowerCase() === 'not in': {
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)